### PR TITLE
Avoid passing incorrect props to HTML

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Emotion fix

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1262** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1262" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1262?utm_source=stack-comment).

File: packages/ui/src/components/Space.tsx [hunk 1]
```diff
@@ -9,7 +9,13 @@ type BaseProps = {
   y?: number | PartialRecord<Level | 'base', number>
 }
 
-export const Space = styled.div<BaseProps>({}, ({ x, y }) => {
+const config = {
+  shouldForwardProp: (prop: string) => !(prop === 'x' || prop === 'y'),
```
Comment (alebedev) [inline packages/ui/src/components/Space.tsx:13]: Would `isValidProp` work here?

Comment (robinandeer) [reply packages/ui/src/components/Space.tsx:13]: I tried but it didn't...

**Review by alebedev — APPROVED** [review]
Looks good

**Pull request merged**